js/src/modals/Verification/verification.js

```diff
@@ -231,20 +231,22 @@ class Verification extends Component {
         } else if (method === 'email') {
           fields.push({
             key: 'email',
-            label: 'email address',
+            label: 'e-mail address',
```

Comment:
Would obviously have preferred FormattedMessage here - it is coming more and more to the fore.